src/zarr/storage/_fsspec.py

```diff
@@ -249,9 +249,8 @@ def from_url(
 
         # fsspec is not consistent about removing the scheme from the path, so check and strip it here
         # https://github.com/fsspec/filesystem_spec/issues/1722
-        if "://" in path and not path.startswith("http"):
-            # `not path.startswith("http")` is a special case for the http filesystem (¯\_(ツ)_/¯)
-            path = fs._strip_protocol(path)
+        # for http(s) and swift, _strip_protocol() does nothing, since they need the protocol in the path
```

Comment:
I don't think the new comment is needed, and the comments above can be removed too.